crates/types/src/ycbcr422_image.rs

```diff
@@ -240,11 +243,3 @@ impl YCbCr422Image {
         })
     }
 }
-
-impl Index<Point2<Pixel, usize>> for YCbCr422Image {
-    type Output = YCbCr422;
-
-    fn index(&self, position: Point2<Pixel, usize>) -> &Self::Output {
-        &self.buffer[position.y() * self.width_422 as usize + position.x()]
-    }
-}
```

Comment:
why opt out of index impl?

Reply:
It was dead code before and it implemented 422 indexing and there is already `.at()` and `.try_at()`. Do you have a suggestion how to proceed? 🤔